sdk/storage/Azure.Storage.Common/src/Shared/StorageClientOptions.cs

```diff
@@ -122,6 +126,33 @@ public static HttpPipeline Build(this ClientOptions options, HttpPipelinePolicy
                 classifier.SecondaryStorageUri = geoRedundantSecondaryStorageUri;
             }
 
+            if (expectContinue != null)
+            {
+                switch (expectContinue.Mode)
+                {
+                    case ExpectContinueMode.ApplyOnThrottle:
+                        pipelineOptions.PerCallPolicies.Add(new ExpectContinueOnThrottlePolicy()
+                        {
+                            ThrottleInterval = expectContinue.ThrottleInterval,
+                            ContentLengthThreshold = expectContinue.ContentLengthThreshold ?? 0,
+                        });
+                        break;
+                    case ExpectContinueMode.On:
+                        pipelineOptions.PerCallPolicies.Add(new ExpectContinuePolicy()
+                        {
+                            ContentLengthThreshold = expectContinue.ContentLengthThreshold ?? 0,
+                        });
+                        break;
+                    case ExpectContinueMode.Off:
+                        break;
+                }
+            }
+            else
+            {
+                // TODO get env config for whether to disable
```

Comment:
Is this going to be done in a separate PR? Or should we wait on the ask for this

Reply:
Yes, this is future PR work. This will be necessary before release but I'm avoiding bloating the PR.